.github/workflows/liquibase-migration-check.yml

```
@@ -37,37 +55,155 @@ jobs:
+        env:
+          CI: true
         run: |
-          echo "Checking for changes in the liquibase changelog..."
+          # Generate changelog diff to detect any schema drift
+          # This creates a changelog_new.xml file if schema changes are detected
+          scripts/db-utils.sh draft-changelog
+
+      - name: Validate database schema and documentation
+        run: |
+          echo "🔍 Validating that database schema matches migrations..."
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)